src/main/java/ch/njol/skript/effects/EffSendBlockChange.java

```diff
@@ -34,84 +32,98 @@
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Effect;
 import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 
 @Name("Send Block Change")
-@Description("Makes a player see a block as something it really isn't. BlockData support is only for MC 1.13+")
-@Examples({"make player see block at player as dirt",
-		"make player see target block as campfire[facing=south]"})
-@Since("2.2-dev37c, 2.5.1 (block data support)")
+@Description("Makes players see blocks as something else.")
+@Examples({
+	"make player see block at player as dirt",
+	"make player see target block as campfire[facing=south]"
+})
+@Since("2.2-dev37c, 2.5.1 (block data support), INSERT VERSION (reset)")
 public class EffSendBlockChange extends Effect {
 
-	private static final boolean BLOCK_DATA_SUPPORT = Skript.classExists("org.bukkit.block.data.BlockData");
-	private static final boolean SUPPORTED =
-			Skript.methodExists(
-					Player.class,
-					"sendBlockChange",
-					Location.class,
-					Material.class,
-					byte.class
-			);
-
 	static {
 		Skript.registerEffect(EffSendBlockChange.class,
-				BLOCK_DATA_SUPPORT ? "make %players% see %blocks% as %itemtype/blockdata%" : "make %players% see %blocks% as %itemtype%"
-		);
+				"make %players% see %blocks% as %itemtype/blockdata%",
+				"make %players% see %blocks% (as normal|the same as the server)",
+				"(reset|sync) %blocks% (for|to) %players% [with the server]");
```

Comment:
i think this pattern's a little awkward and doesn't really add much. The 2nd pattern seems to cover the role very well and matches the format better.